tekton/release-full-setup.md

```diff
@@ -0,0 +1,109 @@
+# Release Pipeline
+
+## Setup
+
+> **Note:**
+> Depending on your cluster you may need to increase the memory allocated. 6GB is known to work on Docker Desktop.
+> See https://github.com/tektoncd/pipeline/issues/2417 for details.
+
+First, ensure that your credentials are set up correctly. You will need an account with access to [Google Cloud Platform](https://console.cloud.google.com). Your account must have 'proper authorization to release the images and yamls' in the [`tekton-releases` GCP project](https://github.com/tektoncd/plumbing#prow). Your account must have `Permission iam.serviceAccountKeys.create`. Contact @bobcatfish or @dlorenc if you are going to be creating dashboard releases and require this authorization.
+
+- You will need to install the [`gcloud` CLI](https://cloud.google.com/sdk/gcloud/) in order to get keys out of Google Cloud and into your local cluster. These credentials will be patched onto the service account to be used by the Tekton PipelineRuns. You do not need to create a new GCP project or pay Google any money.
+- It's convenient to use the Tekton Dashboard to kick off builds, alternatively you can use the ['tkn' CLI](https://github.com/tektoncd/cli) so you may want to grab that as well.
+
+```bash
+KEY_FILE=release.json
+GENERIC_SECRET=release-secret
+PIPELINE_NAMESPACE=tekton-pipelines
+# The kubernetes ServiceAccount that will be used by your Tekton tasks. 'default' is the default. It should already exist.
+SERVICE_ACCOUNT=default
+GCP_ACCOUNT="release-right-meow@tekton-releases.iam.gserviceaccount.com"
+
+# 1. Create a GCP private key for the service account. It is vital to keep a copy safe since there is a limit of ten keys in total.
+gcloud iam service-accounts keys create --iam-account $GCP_ACCOUNT $KEY_FILE
+
+# 2. Sore GCP key in a secret
+kubectl create secret generic $GENERIC_SECRET -n $PIPELINE_NAMESPACE --from-file=./$KEY_FILE
+
+# 3. Patch the GCP key onto the service account to be used to run the release pipeline.
+kubectl patch serviceaccount $SERVICE_ACCOUNT -n $PIPELINE_NAMESPACE -p "{\"secrets\": [{\"name\": \"$GENERIC_SECRET\"}]}"
+```
+
+Next:
+
+1. Install [Tekton pipelines](https://github.com/tektoncd/pipeline) into your local cluster.
+1. Create a GitHub release by pushing a tag to the [dashboard](https://github.com/tektoncd/dashboard) repository. This should be of the form, `vX.Y.Z' e.g.' 'v0.2.5'.
+1. Edit the `tekton-dashboard-git` PipelineResource in `resources.yaml` and set `spec.params.revision.value` to 'vX.Y.Z' e.g., `v0.2.5`. This can also be a git commit if you have not created a release yet.
+1. From the root directory of the dashboard repository, create the Tekton Dashboard release pipeline:
+
+   ```bash
+   PIPELINE_NAMESPACE=tekton-pipelines
+   kubectl apply -f tekton -n $PIPELINE_NAMESPACE
+   ```
```

Comment:
This is still referencing the old pipeline. Replace these with a link to the release cheat sheet so we only maintain those instructions in one place.